lib/Net/Async/Blockchain/BTC.pm

```diff
@@ -192,7 +188,9 @@ async sub hashblock {
     }
 
     my @transactions = map { $_->{block} = $block_response->{height}; $_ } $block_response->{tx}->@*;
-    await Future->needs_all(map { $self->transform_transaction($_) } @transactions);
```

Comment:
Why change this ?
That make async no sense

Reply:
@chilli-binary to reduce the number of concurrent RPC calls to the node.